Tools/autotest/sim_vehicle.py

```diff
@@ -538,7 +538,10 @@ def start_vehicle(binary, autotest, opts, stuff, loc):
     cmd = []
     if opts.valgrind:
         cmd_name += " (valgrind)"
-        cmd.append("valgrind")
+        # adding this option allows valgrind to cope with the overload
+        # of operator new
+        valgrind_opts = "--soname-synonyms=somalloc=nouserintercepts"
```

Comment:
```suggestion
      cmd.append("valgrind")
      cmd.append("--soname-synonyms=somalloc=nouserintercepts")
```

This is the pattern here.  OTOH, given the callgrind stuff immediately below here is mine and doesn't follow the pattern....